package.json

```diff
@@ -50,7 +50,9 @@
     "coveralls": "^2.11.15",
     "enzyme": "^2.6.0",
     "eslint": "^3.10.2",
-    "eslint-config-standard": "^6.2.1",
+    "eslint-config-airbnb": "13.0.0",
+    "eslint-plugin-import": "2.2.0",
+    "eslint-plugin-jsx-a11y": "2.2.3",
     "eslint-plugin-promise": "^3.4.0",
     "eslint-plugin-react": "^6.4.1",
     "eslint-plugin-standard": "^2.0.1",
```

Comment:
Tests fail without this dependency as well. I guess airbnb config relies on this one. I might check for more details.